apps/Standalone/src/mcp/app/McpStandard.tsx

```diff
@@ -16,6 +18,18 @@ export const McpStandard = () => {
     () => WorkflowUtility.convertToCanonicalFormat(workflowAppData?.location ?? 'westus'),
     [workflowAppData]
   );
+  const { data: originalConnectionsData } = useConnectionsData(appId);
```

Comment:
Can you not do this? there is no workflow app context... it is different from template initialization. you also need to remove the workflow app information, there should be different way to initialize connections when workflow app is selected.